dlt/common/utils.py

```diff
@@ -635,3 +635,42 @@ def __getattribute__(self, name: str) -> Any:
                 raise RuntimeError("This instance has been dropped and cannot be used anymore.")
 
     return DefunctClass
+
+
+def obfuscate_string(s: str) -> str:
+    """Obfuscates string by replacing some or all characters with asterisks"""
+    if len(s) < 6:
```

Comment:
I'd consider replacing this magic number with a constant and maybe use a single constant for both this place as well as here: https://github.com/dlt-hub/dlt/blob/c0e4795b2c6854238df7741f03e067fb789ce442/dlt/sources/rest_api/__init__.py#L449

Reply:
actually I think we can get rid of this mask_secrets code entirely, no?